pkg/init/init

```diff
@@ -2,6 +2,14 @@
 
 /bin/mount -t tmpfs tmpfs /mnt
 
+# just copying everything uses a lot of space, so use mv which frees it up
+# copy directory structure under /containers
+find /containers -type d | cpio -p /mnt 2> /dev/null
```

Comment:
Does this read and discard everything while creating the directories? Could be a source of slowness, although given everything is in a tmpfs on both ends perhaps not that likely.